public/docs/getting_started/client.md

```diff
@@ -37,7 +50,7 @@ There are many official and unofficial Ethereum clients available for you to use
 * Parity: [https://github.com/paritytech/parity](https://github.com/paritytech/parity)
 * More: [https://www.ethereum.org/cli](https://www.ethereum.org/cli)
 
-These are full client implementations that include mining, networking, blocks and transaction processing. You should use these clients after you've sufficiently tested your dapp with Truffle Develop or the TestRPC and you're ready to deploy to your desired Ethereum network. Truffle can deploy to these clients without any extra configuration.
+These are full client implementations that include mining, networking, blocks and transaction processing. You should use these clients after you've sufficiently tested your dapp with Ganache or Truffle Develop and you're ready to deploy to your desired Ethereum network. Truffle can deploy to these clients without any extra configuration.
```

Comment:
Noticed that this is slightly wrong: 

> Truffle can deploy to these clients without any extra configuration.

Technically you do need to add configuration steps to deploy to these networks, but you don't need _more than_ what's needed for Ganache or Truffle develop. To remove confusion, we should probably remove or reword that sentence.